tests/e2e/specs/media/insertMediaFromLibrary.js

```diff
@@ -19,10 +19,17 @@
  */
 import { createNewStory } from '../../utils';
 
-describe('Example Spec', () => {
-  it('should be able to create a blank story', async () => {
+describe('Media Library', () => {
+  it('should insert an image from the library after clicking on it', async () => {
     await createNewStory();
 
-    await expect(page).toMatchElement('input[placeholder="Add title"]');
+    // This selector matches all image elements as their class names start with "mediaElement__Container"
+    // See https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors.
+    // Clicking will only act on the first element.
+    await expect(page).toClick('[class|="mediaElement__Image"]');
```

Comment:
I think we should just do some `data-testid` sprinkled initially. In some tests we want to drive more with coordinates, but imho most of time these would be sufficient.

Reply:
Agreed